Misc/NEWS.d/next/Documentation/2019-09-05-14-47-51.bpo-36797.KN9Ga5.rst

```diff
@@ -0,0 +1 @@
+Fix a dead link in the distutils API Reference.
```

Comment:
I think "Patch by Miro Hrončok." is optional, but if you'd like credit in the changelog you can put it here.

Reply:
not really :laughing: